### DISCUSSION TITLE
Show the thread's "latest poster" in overview

### DISCUSSION THREAD

**Discussion by Griatch** [community/community#2951] [Discussions]
As people add to a thread, the threads move up to the top of the list (if sorted by `newest`). This is good. But there is no information as to _who_ actually made this update. In the example below, I have replied to the original poster and we have talked back and forth a few times. But when I get back to this view a little later there is no way to know at a glance if my message is still the last one or if there was a new reply to me, unless I mentally keep track of the number of comments on the right from visit to visit. The avatars on the right only shows who's in the thread as far as I can tell, no further info.

I suggest that there should be something like `[latest post] by Griatch 10m ago` on the thread overview (with a link to that post) once there are >0 replies to a thread. This helps to keep up with changes.

![DeepinScreenshot_select-area_20201211001335](https://user-images.githubusercontent.com/294267/101841191-c216fa00-3b45-11eb-857d-dca09f36f093.png)

**Comment by otseng** [comment]
I would like to see this in both issues and discussions.  We have many threads and I want a way to tell who has posted last.  If a user has posted, then I want to go visit that thread.  If me or another developer posted last, I won't need to visit it.  Showing when the user last posted would also be helpful.

Here's our issues and discussions:
https://github.com/eliranwong/UniqueBible/issues
https://github.com/eliranwong/UniqueBible/discussions

**Reply by onegladiator4u** [reply]
That would be helpful indeed

**Reply by Griatch** [reply]
Yeah, as the number of posts increase, this becomes more and more important.

**Reply by eliranwong** [reply]
Looking forward to it.

**Comment by Griatch** [comment]
This gets more and more important as our usage grows. It's really hard to know who made the latest post and respond to it as a maintainer - especially since that post may be at the end of a nested/hidden thread of the discussion.
